src/io/flutter/vmService/VMServiceManager.java

```diff
@@ -303,13 +308,13 @@ else if (StringUtil.equals(streamId, VmService.LOGGING_STREAM_ID) || StringUtil.
   }
 
   private Object getExtensionValueFromEventJson(String name, String valueFromJson) {
-    final Object enabledValue =
-      ServiceExtensions.toggleableExtensionsWhitelist.get(name).getEnabledValue();
+    final Object sampleValue =
```

Comment:
nit:
sampleValue --> firstValue?

Reply:
alternately better Java style would be to add a
getValueClass() method to the ServiceExtension class that returns a Class object consistent with this.
The class object could be generated from just getting the first value. https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html

Reply:
done